src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -81,7 +81,7 @@ public function getDecoratedFactory()
     /**
      * {@inheritdoc}
      */
-    public function createListFromChoices($choices, $value = null)
+    public function createListFromChoices(iterable $choices, callable $value = null)
```

Comment:
you can't add the callable type to the caching decorator. otherwise we can't decorate the `PropertyAccessDecorator`

Reply:
Oh, yeah, missed that :(